js/cryptopia.js

```diff
@@ -407,7 +407,7 @@ module.exports = class cryptopia extends Exchange {
         //
         // fetchWithdrawals
         //
-        //     {   
+        //     {
```

Comment:
Can you tell me how to do this on my own? I ran the eslint command in the docs but it didnt show anything helpful.

Reply:
Sure, if you have the eslint properly configured in your editor, it can highlight them for you, as shown on the screenshot below (VSCode for code editing):

![vscode](https://user-images.githubusercontent.com/1294454/48724999-e2607d00-ec3b-11e8-8d14-b3612233f258.jpg)

If you don't have it installed into your editor, then you can see those errors in command line with the command you typed (`eslint`), however, if you run it without arguments it will check current directory by default. And it does not show any errors there. So to check for `js/*.js` files you need to supply a wildcard string or a path to it, as shown on the following screenshot (`eslint js/*.js`):

![eslint](https://user-images.githubusercontent.com/1294454/48725160-59961100-ec3c-11e8-90e1-d5d5463fb346.jpg)

And we also have an npm action defined in package.json, so you can simply run:

```
npm run check-js-syntax
```

Which effectively launches the linter and does the same checking (for JS code only).

Reply:
It can also do easy fixes of those typos automatically for you if you run `eslint js/*.js --fix`.

Reply:
@milesalex oh, to clarify: if you're asking why that particular typo didn't show up when you ran the linter – that is because I made the typo after your last edit to the file, so it was introduced by me accidentally and I fixed it right away, so, no worries, no magic here )